spec/lib/user_spec.rb

```diff
@@ -49,12 +49,12 @@
   context 'avoiding duplicate usernames' do
     before do
       srand(2)
-      User.new.generate(email: 'foo@bar.gov.uk')
+      User.new.generate(contact: 'foo@bar.gov.uk')
     end
 
     let!(:user) do
       srand(2)
```

Comment:
What does this srand do?

Reply:
[Seeds the system pseudo-random number generator, Random::DEFAULT, with number. The previous seed value is returned.](http://ruby-doc.org/core-2.0.0/Random.html#method-c-srand)

Reply:
Feels like this User class has many responsibilities.
Would opt for having a password generator class on it's own, but can be a refactoring for the future.